src/Entity/File.php

```diff
@@ -83,44 +84,21 @@ public function getNameView(): ?string
         return $this->nameView;
     }
 
-    public function getNameHash(): ?string
+    public function getCategory(): ?Category
     {
-        return $this->nameHash;
+        return $this->category;
     }
 
-    public function getSubcategory(): ?Subcategory
+    public function getTag(): Collection
```

Comment:
Pamiętaj o liczbie mnogiej ;) Jak potem będziesz korzystać z Twiga to on z kolei korzysta właśnie z getterów, aby dostać się do właściwości encji np:

```
<ul>
{% for tag in file.tags %}
     <li>{{ tag.name }}</li>
{% endfor %}
</ul>
```

file.tags tak naprawdę wywołuje metodę getTags() na obiekcie file